.gitignore

```diff
@@ -4,3 +4,4 @@ build/dev
 docs/*
 !docs/*.conf.json
 !docs/*.ico
+.idea/*
```

Comment:
missing new line. Keep your .idea git ignores in a global gitignore on your own machine.